src/impl/sbcl-avx-vops.lisp

```diff
@@ -0,0 +1,356 @@
+;;;; sbcl-avx-vops.lisp
+;;;;
+;;;; Author: Cole Scott
+;;;;         
+;;;; Collaborators: Jussi Kukkonen
+;;;;                Robert Smith
+
+(in-package #:qvm-intrinsics)
+
+(deftype d4 ()
+  '(simd-pack-256 double-float))
+(deftype cdf ()
+  '(complex double-float))
+
+;;; Function stub definitions
+;;; This tells the compiler about the existance and properties of the VOPs as functions
+
+(defknown (%2x2matrix-to-simd) (cdf cdf cdf cdf)
+    (values d4 d4 d4 d4)
+    (movable flushable always-translatable)
```

Comment:
You claim `movable` here, but `%2x4matrix-to-simd` below does not. Intentional?

Reply:
No. Good catch.